hummingbird/ml/_container.py

```diff
@@ -52,24 +52,57 @@ def __init__(self, input_names, output_names, operator_map, operators, extra_con
             extra_config: Some additional custom configuration parameter
         """
         super(PyTorchBackendModel, self).__init__()
-        self._input_names = input_names
-        self._output_names = output_names
+
+        # Define input \ output names.
+        # This is required because the internal variable names may differ from the original (raw) one.
+        # This may happen, for instance, because we force our internal naming to be unique.
+        self._input_names = []
+        self._output_names = []
+        map = {}
+        for op in operators:
+            for op_input in op.inputs:
```

Comment:
i understand why the first two loops, but is the third loop always necessary?  Is it possible to do the break (line 67) earlier?

Reply:
I tried but I don't it is possible.